content/en/news/releases/1.15.x/announcing-1.15.2/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.15.1
+linktitle: 1.15.1
+subtitle: Patch Release
+description: Istio 1.15.1 patch release.
+publishdate: 2022-09-26
+release: 1.15.1
+---
+
+This release includes security fixes in Go 1.19.2 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
+
+This release contains bug fixes to improve robustness.
+
```

Comment:
Need to verify if we want these lines as separate lines. In earlier release notes, we haven't (ex: https://istio.io/latest/news/releases/1.14.x/announcing-1.14.3/)